src/app/api/core/services/policies.service.ts

```diff
@@ -16,6 +16,7 @@ export class PoliciesService extends BaseService {
       [Resource.ViewSetting]: [],
       [Resource.Users]: [],
       [Resource.Attachments]: [UserAction.Read],
+      [Resource.Comment]: [UserAction.Create, UserAction.Read, UserAction.Update],
```

Comment:
Can we please confirm that client users can also add / read / update comments?

Reply:
Yeah, will confirm in tomorrow's call. For now I am removing this to allow readonly.